R/pkg/R/SQLContext.R

```diff
@@ -172,10 +221,10 @@ getDefaultSqlSource <- function() {
 createDataFrame <- function(data, schema = NULL, samplingRatio = 1.0,
                             numPartitions = NULL) {
   sparkSession <- getSparkSession()
-
+  arrowEnabled <- sparkR.conf("spark.sql.execution.arrow.enabled")[[1]] == "true"
```

Comment:
is it always in the conf - I think you can also pass in a default value to sparkR.conf

Reply:
Yea,I checked that it always has the default value. I initially left the default value but took it out after double checking.

Reply:
ok